### ISSUE TITLE
Vorschlag: Burgruinen sollten stehen bleiben, nachdem die Ki besiegt wurde.

### ISSUE THREAD

**Issue by Fufudrache** [Sh0wdown/UnofficialCrusaderPatch#191]
Hi, 

mich hat es als Kind immer gestört, dass, wenn man die Ki besiegt hat ihre Burgreste verschwinden und nur der Burggraben übrig bleibt. Ich fände es cool, wenn die Burgreste einfach stehen bleiben würde und von den anderen Spielern genutzt werden könnten (sofern die Treppen etc. nicht zerstört wurden). Mich faszinieren so Ruinen einfach :D 

Alternativ könnte man zumindest machen, dass die Burggräben mit der restlichen Burg verschwinden. Das stört mich immer so arg! :D 

Wollte das nur mal in die Runde werfen :D 

Viele Grüße und so!

**Comment by Monsterfisch** [comment]
Ruinen bitte! Häuser und Werkstätten könnten dann auch zu den unzerstörbaren Editor Ruinen werden  (vll zerstörbar machen?). So siehts schön verlassen aus und stellt weiterhin eine Art Hinderniss bzw eine Art Schutz zur Verfügung

**Comment by Monsterfisch** [comment]
Ich hatte schon mit dem gedanken gespielt die besiegten gegner auf die seite des Sieger zu wechseln (neue start resourcen und alles geben um wieder mitmischen zu können und vielleicht nur ne % chance machen bei bestimmten charakteren schlange 90% wolf 10% und scheriff so 50% um überzulaufen). kann mir da aber auch gut vorstellen das das match schnell sehr einseitig wird bzw ein gegner immer und immer wieder die seite wechselt ;)

**Comment by Heroesflorian** [comment]
> Ich hatte schon mit dem gedanken gespielt die besiegten gegner auf die seite des Sieger zu wechseln (neue start resourcen und alles geben um wieder mitmischen zu können und vielleicht nur ne % chance machen bei bestimmten charakteren schlange 90% wolf 10% und scheriff so 50% um überzulaufen). kann mir da aber auch gut vorstellen das das match schnell sehr einseitig wird bzw ein gegner immer und immer wieder die seite wechselt ;)

Ja, das birgt dann vermutlich wirklich die Gefahr, sehr einseitig zu werden, oder zu "Endlospartien" zu führen, wenn besiegte Gegner als kompletter Spieler wieder "respawnen"... 
Und wirft auch einige weitere Fragen auf, z.B. wie das bei menschlichen Spielern funktionieren würde: Kann man in Skirmish-Partien mit Verbündeten dann auch die Seite wechseln? (Yay Kreuzzugsmarsch easy!)
Und kann man durch Lord-vorschicken einfach wieder zurückwechseln - oder überhaupt einfach mal Lord rausschicken, um "präventiv" mal die Seite zu wechseln, statt zerbombt zu werden? 
Kann man dann überhaupt eine Partie noch verliere, denn am Ende wären ja immer alle auf der Gewinnerseite? (Ja, nicht immer, wenn es nur eine gewisse %-Chance zum Seitenwechsel gibt, aber zumindest "oftmals" / "deutlich öfter als nie".)
Und wie wäre das im Mehrspieler... 

An sich wärs natürlich interessant, aber vermutlich nochmal deutlich schwieriger und komplexer als eine Art Dorf/Außenposten, der noch "irgendwas" macht (seien es Steuertribute, eigene Truppen (ähnlich wie die Extreme-Außenposten nur nicht so unrealistisch und übertrieben), oder was auch immer). 

Und ganz prinzipiell fände ich ja auch eine dynamischere "Diplomatie" interessant - mit möglichem Seitenwechsel, bevor jemand stirbt... aber das ist dann wohl noch eine andere Geschichte als die Ruinen hier^^

**Comment by Monsterfisch** [comment]
Prinziepiell würde ich das überlaufen nur den AIs möglich machen.
Wäre zwar schon köstlich wenn jemand einfach seinen Lord zu den Gegnern schickt um überzulaufen so richtig schön link in den Rücken fallen(rip Freundschaften) :D 
das Ländereien System das in sh2 und crusader2 vorhanden ist wäre natürlich auch super  aber dafür müsste man dann eine spezielle "dorf" ki hinzufügen  die Truppen und Resourcen produziert und dann dem Eroberer schickt. (man nehme hier theoretisch die Ratte zur hand als platzhalter. Ffrage ist dann wird das dorf dann immer den Eroberern gehören oder kann das wiederum von den feinden geklaut werden und wie oft und mit welcher Priorität. Ich denke im reinen Gedankenspielkonzept wäre es schlichtweg einfacher die ki wieder mit einer % chance (mit jedem mal sterben sinkt der Prozentsatz) mal wieder einsteigen zu lassen damits nicht unendlich oft passiert und wenn er halt keinen bock mehr hat gemobbt zu werden lässt er halt ne ruine zurück  ;)

**Comment by Panbutt** [comment]
Yes, I love the idea of ruins! Maybe when a Lord is killed, every castle asset (walls, towers, gates) could randomly loose anywhere between 25-75% hp? That way some walls, gates and towers would be destroyed. I think industry buildings should remain all destroyed. But I would love if towers, gates and walls remained. 

I also think traps like dog cages, kill pits and pitch should remain as well. Giving castle ruins a little sense of danger to them still. Maybe all active siege engines of the AI could also be abandoned? Allowing the player to recover them if possible.

Not sure how complex all that would be to implement however ..

**Comment by Halbarad1** [comment]
Would be really nice if ruins would stay. Nice suggestion :)
